websites/A/AnimeDigitalNetwork/presence.ts

```diff
@@ -47,7 +46,7 @@ presence.on("UpdateData", async () => {
         )
         .textContent,
       data = {
-        details: "Browsing...",
+        details: "Looking at...",
```

Comment:
well I literally requested changes without these 3 dots. Remove them.
and im not `riri` ty